styleguide/derek/pattern-components/ctas/cta-bottom.ejs

```diff
@@ -8,7 +8,7 @@
     </div>
     <div class="cta-row">
       <div class="cta-buttonWrapper">
-        <a class="cta-button">Talk to a Specialist</a>
+        <a class="cta-button"  event-track>Talk to a Specialist</a>
```

Comment:
This needs to be class `.track-cta` - https://github.com/rackerlabs/zoolander/blob/56a2afe1eb38973f30df7a9eb07759ffee5195c5/styleguide/derek/global-components/tracking/tracking.spec.js#L219